doc/man5/config.pod

```diff
@@ -64,8 +64,8 @@ length after variable expansion. Otherwise an error will occur.
 
 It is possible to escape certain characters by using any kind of quote
 or the B<\> character. By making the last character of a line a B<\>
-a B<value> string can be spread across multiple lines. In addition
-the sequences B<\n>, B<\r>, B<\b> and B<\t> are recognized.
+a B<value> string can be spread across multiple lines. In addition, a
+B<\> can be followed by one of B<n>, B<r>, B<b>, or B<t>.
```

Comment:
I'm not sure that this particular change was for the better...

Reply:
Well, backslash-char means special things to pod and man.  And it also describes the way the code works better.  Still not okay?

Reply:
I don't see how that describes things better. Regarding backslashes, they have no general special meaning in the pod format (see `man perlpod`). `pod2man` converts them to `\e`, so roff converters will display them correctly.

Did you try `perl doc doc/man5/config.pod`? If you want to see the roff produced, you can give it `-n less`. That's a good way to check your claims...